cmd/operator/main.go

```diff
@@ -156,7 +160,9 @@ func main() {
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme: scheme,
 		Metrics: metricsserver.Options{
-			BindAddress: fmt.Sprintf("%s:%d", metrics.Host, metrics.Port),
```

Comment:
Why `9182` port is no longer relevant?

https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/metrics/metrics.go#L46